frontend/src/scenes/experiments/experimentsLogic.ts

```diff
@@ -55,7 +55,7 @@ export const experimentsLogic = kea<experimentsLogicType>({
     }),
     events: ({ actions }) => ({
         afterMount: () => {
-            actions.loadExperiments()
+            actions.loadExperiments(toParams({ all: true }))
```

Comment:
what's this all true params for?

Reply:
The one you added for tabs 😅 . Without it, on first load, archived experiments show up in the main list, but when you click on "All Experiments", they vanish

Reply:
HAHA

Reply:
tabs life